src/main/java/frc/robot/subsystems/loader/SetLoaderVelocity.java

```diff
@@ -29,26 +34,42 @@ public SetLoaderVelocity(double desiredVelocity) {
         feedforward = robotConstants.controlConstants.loaderFeedforward;
         pidController = new TrigonPIDController(robotConstants.controlConstants.loaderPidSettings, desiredVelocity);
     }
+
     /**
      * This class accelerates the loader subsystem to the desired velocity using PID.
      * This constructor is used for tuning PID using the dashboard
      */
-
     public SetLoaderVelocity() {
         addRequirements(loader);
         feedforward = robotConstants.controlConstants.loaderFeedforward;
-        pidController = new TrigonPIDController("Loader pid controller");
+        pidController = new TrigonPIDController("Loader PID controller");
     }
 
     @Override
     public void initialize() {
         pidController.reset();
+        inStall = false;
+        firstStallTimestamp = 0;
+        initializeTimestamp = Timer.getFPGATimestamp();
     }
 
     @Override
     public void execute() {
-        loader.setVoltage(
-            pidController.calculate(loader.getVelocity(), -6, 6) + feedforward.calculate(pidController.getSetpoint()));
+        if (Timer.getFPGATimestamp() - firstStallTimestamp < robotConstants.loaderConstants.kSpinBackwardsTime) {
+            loader.move(-feedforward.calculate(pidController.getSetpoint()));
+            inStall = false;
+            return;
+        }
+        if (!inStall && loader.getIsInStall() && Timer.getFPGATimestamp() - initializeTimestamp >
+            robotConstants.loaderConstants.kTimeout) {
+            DriverStationLogger.logToDS("A cell got stuck in the loader. Trying to unjam it now");
```

Comment:
DriverStationLogger.logToDS("A cell got stuck in the loader. Trying to unjam it now");
what is **unjam**? and remove **now**

Reply:
[Here is defenition](https://www.yourdictionary.com/unjam)
It basically means that it's making it not stuck